### ISSUE TITLE
When clicking "Upload a file" in audio tab when editing a card game, existing audio is removed

### ISSUE THREAD

**Issue by johnnynotsolucky** [ji-devs/ji-cloud#2549]
**Describe the bug**

When clicking "Upload a file" in audio tab when editing a card game, existing audio is removed without prompting the teacher if they want to remove it.

**To Reproduce**
1. Create a card game and add audio to a card;
2. Once audio is saved on the card, click the "Upload a file" button;
3. Note that the uploaded audio is removed, and you need to click "Upload a file" again to switch to the upload view.

**Expected behavior**

The teacher should be prompted if they want to remove the existing audio, if there is audio. Otherwise, they should be able to switch without a prompt.

**Screenshots**

#### Audio recorded and saved

![Screenshot 2022-03-15 at 10 33 11](https://user-images.githubusercontent.com/4161106/158338127-930d2fe0-6445-45a4-be18-9e159d411b39.png)

#### Clicked "Upload a file"

![Screenshot 2022-03-15 at 10 33 19](https://user-images.githubusercontent.com/4161106/158338195-a3e6edcf-2d0a-470c-bfe5-543e1bdf4401.png)

**Comment by corinnewo** [comment]
Good catch! We need a warning here. @ChaykeeMor please edit: "Wait! Uploading a sound file will remove the sound that is in place. Are you sure?"

**Comment by ChaykeeMor** [comment]
"Are you sure? Uploading a file replaces your recording." Continue/Cancel

**Comment by ChaykeeMor** [comment]
Not sure if this should be here or have it's own ticket...but the 'add sound' button [should be 'add audio'] reverts to record. Should open browse window like the large upload box does. @corinnewo 

<img width="329" alt="Screen Shot 2022-09-01 at 00 07 39" src="https://user-images.githubusercontent.com/94056434/187863111-6be1d48c-2550-4b87-8d17-ffc0e141bbeb.png">
